### PR TITLE
Fixed issue [Spring]Add support for the contextPath in Spring-boot gen #3193

### PR DESCRIPTION
**Pull request by vovan-**
Fixed issue [Spring]Add support for the contextPath in Spring-boot gen #3193

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/swagger-api/swagger-codegen/graphs/contributors.

Let me know if you need help fixing it.

Ref: http://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit

**Comment by vovan-** [comment]
Linked commit to my Github account, thank you

**Comment by wing328** [comment]
cc @cbornet

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ public void processOpts() {
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "HomeController.java"));
                 supportingFiles.add(new SupportingFile("swagger2SpringBoot.mustache",
                         (sourceFolder + File.separator + basePackage).replace(".", java.io.File.separator), "Swagger2SpringBoot.java"));
-                supportingFiles.add(new SupportingFile("application.properties",
+                supportingFiles.add(new SupportingFile("application.mustache",
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java]: @vovan- can you please also do the same for `spring-mvc`?

https://github.com/swagger-api/swagger-codegen/pull/3307/files#diff-8642abd39c9b8b007a73f66dad7250caR139

Comment (vovan-) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java]: For Spring mvc property 'server.contextPath' means nothing, it is spring-boot related as noted here http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
So is there a reason to use same application.properties file for Spring boot and Spring mvc?

Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java]: I see. Didn't realise it's for spring boot only. I'll merge the PR shortly.

**Comment by wing328** [comment]
@vovan- PR merged. Thanks for the contribution.

**Comment by cbornet** [comment]
The old "application.properties" should have been deleted !

**Comment by vovan-** [comment]
Old application.properties are still in use by other type of project - Spring MVC

**Comment by cbornet** [comment]
Oh, I thought it was only renaming.
Then the application.xxx files should both be named application.mustache and be put in the respective libraries dir.

**Pull request merged**